onfido-java/src/test/java/com/onfido/integration/WebhookManagerTest.java

```diff
@@ -107,4 +109,18 @@ public void listWebhooksTest() throws Exception {
     assertEquals("https://example.com/firstWebhook", webhooks.get(0).getUrl());
     assertEquals("https://example.com/secondWebhook", webhooks.get(1).getUrl());
   }
+
+  @Test
+  public void resendWebhooksTest() throws Exception {
+    prepareMock("", null, 204);
+
+    onfido.webhook.resend(
+      WebhookResend.request().data(
+        WebhookResendObject.request().resourceType("check").resourceId("check_id_1").event("check_completed"),
```

Comment:
It's just an example/test data, but shouldn't be `check.completed` instead?

That's at least what we allow [here](https://github.com/onfido/onfido-openapi-spec/blob/master/generated/artifacts/openapi-yaml/openapi/openapi.yaml#L4148-L4152).